src/main/java/com/google/devtools/build/lib/packages/ImplicitOutputsFunction.java

```diff
@@ -161,7 +173,7 @@ public SkylarkImplicitOutputsFunctionWithMap(ImmutableMap<String, String> output
    */
   public abstract static class SafeImplicitOutputsFunction extends ImplicitOutputsFunction {
     @Override
-    public abstract Iterable<String> getImplicitOutputs(AttributeMap map);
+    public abstract Object getImplicitOutputs(AttributeMap map); /* guaranteed to output Iterable<String> */
```

Comment:
Apologies, this comment is generally incorrect and was from my own development - it corresponded to the behavior elsewhere that performs reflection tests where I was only able to test for `Iterable`, and not `Iterable<String>`, to assure myself (and others) that the only possible generic for the `Iterable` was `String`.  I can clarify or remove it